### PR TITLE
feat(browsingContext): implement `setViewport`

### PR DESCRIPTION
**Pull request by OrKoN**
Bug: #868
Spec: https://w3c.github.io/webdriver-bidi/#command-browsingContext-setViewport
Delegating edge-cases and extensive validation to WPT tests: https://github.com/web-platform-tests/wpt/pull/40584

### CODE REVIEW

**Comment by thiagowfx** [comment]
I created a new commit on top of this with a few adjustments. Implementation LGTM.

I just wonder if we should add more E2E tests.

**Comment by thiagowfx** [comment]
Ah yes, we should add at least one more test, to exercise this part of the spec:

> If the width or height is outside of the implementation-specific or runtime constraints, return an [error](https://w3c.github.io/webdriver/#dfn-errors) with the [error code](https://w3c.github.io/webdriver/#dfn-error-code) [unsupported operation](https://w3c.github.io/webdriver/#dfn-unsupported-operation).

And maybe another one to test non-top-level contexts.

**Comment by thiagowfx** [comment]
Synced off-line with @OrKoN: He will continue this PR. Edited the comment above with some TODOs once you continue it.

File: tests/test_viewport.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    result = await execute_command(
+        websocket, {
+            "method": "script.evaluate",
+            "params": {
+                "expression": "({'width': window.innerWidth, 'height': window.innerHeight})",
+                "target": {
+                    "context": context_id
+                },
+                "resultOwnership": "root",
```
Comment (sadym-chromium) [inline tests/test_viewport.py:45]: NIT: In this way there will be no `handle` in the result.
```suggestion
                "resultOwnership": "none",
```

**Review by sadym-chromium — APPROVED** [review]
Consider adding (optional) changes to e2e

**Comment by OrKoN** [comment]
I have overlooked the e2e test suggestions, will open a follow-up for the unsupported operation (I am not sure if we have a limit in Chrome) and the rest should be covered by WPT.

**Comment by thiagowfx** [comment]
LGTM
